### PR TITLE
Fix gppylib/operations unit tests 

### PR DESCRIPTION
**Pull request by Annu149**
- tests from gpMgmt/bin/gppylib/operations dir  were not being executed with 'make check' as __init__.py file was missing from gppylib/operations/test directory, added the file.

- Fixed few of the failing tests

## Here are some reminders before you submit the pull request
- [ ] Add tests for the change
- [ ] Document changes
- [ ] Communicate in the mailing list if needed
- [ ] Pass `make installcheck`
- [ ] Review a PR in return to support the community

### CODE REVIEW

File: gpMgmt/bin/gppylib/operations/test/unit/test_unit_segment_reconfigurer.py [hunk 1]
```diff
@@ -43,20 +43,26 @@ def setUp(self):
     def test_it_triggers_fts_probe(self):
         reconfigurer = SegmentReconfigurer(logger=self.logger,
                 worker_pool=self.worker_pool, timeout=self.timeout)
-        reconfigurer.reconfigure()
-        psycopg2.connect.assert_has_calls([
-            call(dbname=self.db, host=self.host, port=self.port, options=None, user=self.user, password=self.passwd),
-            call().query(FTS_PROBE_QUERY),
-            call().close(),
-            ]
+        reconfigurer._trigger_fts_probe(self.db_url)
```
Comment (jnihal) [inline gpMgmt/bin/gppylib/operations/test/unit/test_unit_segment_reconfigurer.py:46]: I believe this step was asserting whether the `reconfigurer.reconfigure()` call was triggering FTS probe or not. By manually calling `reconfigurer._trigger_fts_probe(self.db_url)`, we are creating a false positive.

File: gpMgmt/bin/gppylib/operations/test/unit/test_unit_buildMirrorSegments.py [hunk 2]
```diff
@@ -477,7 +477,7 @@ def test_clean_up_failed_segments(self):
         build_mirrors_obj._clean_up_failed_segments()
 
         self.mock_get_segments_by_hostname.assert_called_once_with([failed1, failed3])
-        self.mock_logger.info.called_once_with('Cleaning files from 3 segment(s)')
+        self.mock_logger.info.assert_called_once_with('Cleaning files from 2 segment(s)')
```
Comment (jnihal) [inline gpMgmt/bin/gppylib/operations/test/unit/test_unit_buildMirrorSegments.py:480]: Why the change in the number of segments? Is it due to the new cleanup of progress files PR?

Comment (Annu149) [reply gpMgmt/bin/gppylib/operations/test/unit/test_unit_buildMirrorSegments.py:480]: According to function 
```
  def _clean_up_failed_segments(self):
        segments_to_clean_up = []
        for toRecover in self.__mirrorsToBuild:
            is_in_place = toRecover.getFailedSegment() is not None and toRecover.getFailoverSegment() is None
            if is_in_place and toRecover.isFullSynchronization():
                segments_to_clean_up.append(toRecover.getFailedSegment())`
```
                
   Files will be cleaned up for segments going under full in-place recovery and in this testcase we have 2 such segments inplace_full1, inplace_full1
   ```
inplace_full1 = GpMirrorToBuild(failed1, live1, None, True, False)
        not_inplace_full = GpMirrorToBuild(failed2, live2, failover2, True, False)
        inplace_full1 = GpMirrorToBuild(failed3, live3, None, True, False)
        inplace_not_full = GpMirrorToBuild(failed4, live4, None, False, False)
        inplace_differential = GpMirrorToBuild(failed5, live5, None, False, True)

        build_mirrors_obj = GpMirrorListToBuild(
            toBuild=[inplace_full1, not_inplace_full, inplace_full2, inplace_not_full, inplace_differential],
            pool=None,
            quiet=True,
            parallelDegree=0,
            logger=self.mock_logger,
            forceoverwrite=True
        )
```

File: gpMgmt/bin/gppylib/operations/test/unit/test_unit_segment_reconfigurer.py (hunk 1 above)
Comment (Annu149) [reply gpMgmt/bin/gppylib/operations/test/unit/test_unit_segment_reconfigurer.py]: Updated
